www-client/seamonkey/seamonkey-2.53.10.2.ebuild

```
@@ -0,0 +1,557 @@
+	# of --build, and --target intstead of --host.
+	# Note, mozilla also has --build but it does not do what you think it does.
+	# Set both --target and --host as mozilla uses python to guess values otherwise
+	mozconfig_annotate '' --target="${CHOST}"
+	mozconfig_annotate '' --host="${CBUILD:-${CHOST}}"
+
+	mozconfig_use_enable pulseaudio
+	# force the deprecated alsa sound code if pulseaudio is disabled
+	if use kernel_linux && ! use pulseaudio ; then
+		mozconfig_annotate '-pulseaudio' --enable-alsa
+	fi
+
+	# For testing purpose only
+	mozconfig_annotate 'Sandbox' --enable-content-sandbox
+
+	mozconfig_use_enable system-sqlite
+	mozconfig_use_with system-jpeg
+	mozconfig_use_with system-icu
+	mozconfig_use_with system-libvpx
+	mozconfig_use_with system-png
+	mozconfig_use_with system-harfbuzz
+	mozconfig_use_with system-harfbuzz system-graphite2
+	mozconfig_use_with system-av1
+
+	# Modifications to better support ARM, bug 553364
+	if use cpu_flags_arm_neon ; then
+		mozconfig_annotate '' --with-fpu=neon
+		mozconfig_annotate '' --with-thumb=yes
+		mozconfig_annotate '' --with-thumb-interwork=no
+	fi
+	if [[ ${CHOST} == armv* ]] ; then
+		mozconfig_annotate '' --with-float-abi=hard
+		if ! use system-libvpx ; then
+			sed -i -e "s|softfp|hard|" media/libvpx/moz.build || die
+		fi
+	fi
+
+	if use lto ; then
+		# Linking only works when using ld.gold when LTO is enabled
+		mozconfig_annotate "forcing ld=gold due to USE=lto" --enable-linker=gold
+		# ThinLTO is currently broken, see bmo#1644409
+		mozconfig_annotate '+lto' --enable-lto=full
+	else
+		if tc-ld-is-gold ; then
+			mozconfig_annotate "linker is set to gold" --enable-linker=gold
+		else
+			mozconfig_annotate "linker is set to bfd" --enable-linker=bfd
+		fi
+	fi
+	# LTO flag was handled via configure
+	filter-flags '-flto*'
+
+	##################################
+	# Former mozconfig_config() end  #
+	##################################
+
+	# enable JACK, bug 600002
+	mozconfig_use_enable jack
+
+	# It doesn't compile on alpha without this LDFLAGS
+	use alpha && append-ldflags "-Wl,--no-relax"
+
+	# Linking fails without this due to memory exhaustion
+	use x86 && append-ldflags "-Wl,--no-keep-memory"
+
+	if ! use roaming ; then
+		MEXTENSIONS+=",-sroaming"
+	fi
+
+	# Setup api key for location services
+	printf '%s' "${_google_api_key}" > "${S}"/google-api-key
+	mozconfig_annotate '' --with-google-location-service-api-keyfile="${S}/google-api-key"
+	mozconfig_annotate '' --with-google-safebrowsing-api-keyfile="${S}/google-api-key"
+
+	mozconfig_annotate '' --enable-extensions="${MEXTENSIONS}"
+	mozconfig_use_enable chatzilla irc
+	mozconfig_annotate '' --enable-dominspector
+
+	# use startup-cache for faster startup time
+	mozconfig_annotate '' --enable-startupcache
+
+	# Broken on some arches
+	mozconfig_annotate '' --disable-elf-hack
+
+	# Use an objdir to keep things organized.
+	echo "mk_add_options MOZ_OBJDIR=${BUILD_OBJ_DIR}" >> "${S}"/.mozconfig
```

External commands which may fail should have a `|| die` at the end, e.g.:
```suggestion
	echo "mk_add_options MOZ_OBJDIR=${BUILD_OBJ_DIR}" >> "${S}"/.mozconfig || die
```

Same for other echos.